components/brave_wallet/browser/blockchain_list_parser.cc

```diff
@@ -266,4 +336,243 @@ bool ParseChainList(const std::string& json, ChainList* result) {
   return true;
 }
 
+bool ParseDappLists(const std::string& json, DappListMap* dapp_list) {
```

Comment:
Are both of these values coming from the component updater? Just want to double check what the trust level is on these.

Reply:
I think the json string should be sanitised first with a `json::convert_all_numbers_to_string()` before passing it to `ParseDappLists`.

Reply:
I see you already sanitize with `data_decoder::JsonSanitizer::Sanitize`, but I think you still need the above for safely parsing the number fields into integer/double types.

Reply:
This seems like a security blocking issue so I'm going to dismiss my original review until this is addressed.